controllers/TransportTicketController.js

```diff
@@ -0,0 +1,87 @@
+const {Transport_ticket} = require('./../database.js');
```

Comment:
Lets use simple name for model "Ticket"

P.S. Use camel case for multi-wording names